python/ccxt/base/decimal_to_precision.py

```diff
@@ -164,6 +164,8 @@ def power_of_10(x):
 
 def number_to_string(x):
     # avoids scientific notation for too large and too small numbers
+    if x is None:
```

Comment:
Why do we add it in Py, but not in JS and not in PHP ?

Reply:
we added it there cause it breaks in py according to a user report

Reply:
i'll add it to all the langs tho